.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Issue report
+about: Report an issue to help us improve the devguide
+---
+
+<!--
+Note: This repo is for the Python devguide. If you are reporting an issue or
+enhancement for the standard library, then the CPython issue tracker is better
+suited for this report: https://bugs.python.org
+
+Before filing a new issue, please search the issues to see if there is already
+a similar issue. If there is, please add a comment to the existing open issue.
+-->
+
+**Describe the issue**
+
+Please provide more information to help us understand the issue:
+
+* Are you reporting a bug, or opening a new enhancement request?
```

Comment:
Would it be worth having an enhancement template and a bug template instead of trying to have a single one handle everything?

Reply:
I think that makes good sense to me.  @Mariatta (GH template guru), thoughts?